components/button/Button.stories.tsx

```diff
@@ -8,3 +9,4 @@ export default {
 
 export const startgame = () => StartGameBtn();
 export const rules = () => RulesBtn();
+export const back = () => BackBtn();
```

Comment:
I would use JSX here if you call React components.

```suggestion
export const back = () => <BackBtn />;
```

In addition, I would make use of StoryBooks Template concept:
https://storybook.js.org/docs/react/get-started/whats-a-story